pkg/operator/resource/resourceapply/core.go

```diff
@@ -168,17 +169,28 @@ func ApplyConfigMap(client coreclientv1.ConfigMapsGetter, recorder events.Record
 			modifiedKeys = append(modifiedKeys, "data."+existingCopyKey)
 		}
 	}
+	for existingCopyKey, existingCopyBinValue := range existingCopy.BinaryData {
+		if requiredBinValue, ok := required.BinaryData[existingCopyKey]; !ok || !bytes.Equal(existingCopyBinValue, requiredBinValue) {
```

Comment:
isn't binary data and data normalized to eachother by the apiserver?

Reply:
no thats `stringData` in Secrets